files/k3s-install.sh

```diff
@@ -11,6 +11,7 @@ hostnamectl set-hostname kubernetes
 sed -i 's/localhost$/localhost kubernetes/' /etc/hosts
 
 ln -sf /run/systemd/resolve/resolv.conf /etc/resolv.conf
+echo "options single-request" >> /run/systemd/resolve/resolv.conf
```

Comment:
Doesn't systemd-resolved overwrite these changes? Or is that only for `/etc/resolv.conf`?

Reply:
oh yeah, ofc it does